client/lib/geocoding/test/index.js

```diff
@@ -1,14 +1,9 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-
-/**
- * Internal dependencies
- */
 import { geocode, reverseGeocode } from '../';
 
-jest.mock( '@automattic/load-script', () => require( './mocks/load-script' ) );
+jest.mock( '@automattic/load-script', () => require( './mocks/load-script' ).default );
```

Comment:
This was throwing `UnhandledPromiseRejectionWarning: TypeError: (0 , _loadScript.loadScript) is not a function`